webcompat/static/js/lib/issues.js

```diff
@@ -484,6 +485,10 @@ issues.MainView = Backbone.View.extend({
   },
   render: function() {
     this.$el.fadeIn();
+  },
+
+  handleKeyShortcuts: function(e) {
+    Mousetrap.bind('mod + enter', _.bind(this.addNewComment, this));
```

Comment:
A few more things, then this is ready!

eslint will complain about single quotes, so can you change to use double quotes? Also, there's an `e` argument getting passed in that isn't being used.

Also, for me it's not working as `mod + enter`, but does work without the spaces `mod+enter'. Does the spaces variant work for you?